codestodians/food-basket-front-end
# [Bug: #163762582] Protect selected routes

#### What does this PR do?
- Creates a Protected route wrapper around Protected components

#### Description of Task to be completed?
- Create protected route wrapper
- Mount Profile component in Protected wrapper.

#### How should this be manually tested?
- Try to visit `/profile` without logging in. The app no longer errors. User is redirected to landing page.

#### What are the relevant pivotal tracker stories?
- [#163762582](https://www.pivotaltracker.com/story/show/163762582)
